docs/index.md

```diff
@@ -1,4 +1,164 @@
+```
+## Querying the metadata body
+The metadata itself is stored in the jsonb format, for help with the syntax of querying jsonb check out this [cheat sheet](https://hackernoon.com/how-to-query-jsonb-beginner-sheet-cheat-4da3aa5082a3).
+
+Get a list of submissions for a particular submitter
+```postgresql
+SELECT p.uuid, p.body->'project_core'->>'project_title' AS title
+FROM project AS p
+WHERE p.body @> '{"contributors": [{"contact_name": "Aviv,,Regev"}]}'
+   OR p.body @> '{"contributors": [{"contact_name": "Sarah,,Teichmann"}]}';
+```
+
+## Querying the experimental graph
+If you have a fastq file and you are curious about its inputs you can use a custom sql function `get_all_parents` to get all of the processes that led to its creation. For example:
+
+If your sequence file has a uuid of `0b978294-b297-4a2f-bc2a-b5b67ab1f71e` you can run 
+```postgresql
+SELECT * FROM process_file_join_table 
+WHERE file_uuid = '0b978294-b297-4a2f-bc2a-b5b67ab1f71e';
+```
+This should return a list of all the processes the file is associated with. Find a row that says `OUTPUT_ENTITY` and copy the `process_uuid`
+
+Then run the following to get a list of all of its parent processes
+```postgresql
+SELECT * FROM  get_all_parents('f7755994-8ba5-4d79-b9fb-7274f0dbfc52')
+```
+- Direct parents are processes whose output becomes the input for the current process
+- `get_all_parents` grabs the processes whose output was the input for the current process and then gets the processes whose output was the input for the parent process and then goes all the way up to the initial file (typically a `donor_organism` file)
+- You can also use the `get_all_children` function to go the opposite way
+- This allows us to represent graphical data in a relational database
+
+`donor_organism` files are typically at the top of the graph. If you have a `file_uuid` for a `donor_organism` and you'd like to get all of the `sequence_files` that came from that donor you can run the following substituting in the `file_uuid` of your `donor_organism`.
+```postgresql
+SELECT *
+FROM (SELECT file_uuid
+      FROM process_file_join_table
+      WHERE process_uuid IN (SELECT get_all_children(process_uuid)
+                             FROM process_file_join_table
+                             WHERE file_uuid = 'c9e7f3d4-c2ce-4e88-ac86-339847d33a6b')) AS temp_table
+       JOIN files ON temp_table.file_uuid = files.uuid
+WHERE files.dcp_schema_type_name = 'sequence_file';
+```
+Further notes on the above query:
+
+- To get all file types, leave off the final `WHERE` clause
+- To retrieve files higher in the graph, replace `get_all_children` with `get_all_parents` 
+
+If you are trying to get all of the files in a graph, it is necessary to specifically retrieve:
+- Any input files and protocol files for a process when running `get_all_children` 
+- Any  output files and protocol files for a process when running `get_all_parents`
```

Comment:
Please remove the extra space after "Any".

Reply:
👍